tests/test_flatten.py

```diff
@@ -0,0 +1,36 @@
+from unittest.mock import patch
+
+from geopandas import GeoDataFrame
+from pandas import DataFrame
+from pytest import raises
+from shapely.geometry import Point
+
+from crossfire import flatten
+
+DICT_DATA = [
+    {"answer": 42, "contextInfo": {"context1": "info1", "context2": "info2"}}
+]
+PD_DATA = DataFrame(DICT_DATA)
+geometry = [Point(4, 2)]
+GEOPD_DATA = GeoDataFrame(DICT_DATA, crs="EPSG:4326", geometry=geometry)
+
+
+# test the existence of the flatten function
+assert flatten
+
+
+def teste_flatten_wrong_nested_columns_value_error():
+    with raises(ValueError):
+        flatten(DICT_DATA, nested_columns="wrong")
+
+
+# test the flatten function with a dictionary mocking it to assert _flatten_dict function is being called
```

Comment:
This makes no sense. The best way to make sure `_flatten_dict` is calling it and expecting the return value to be the data transformed as planned.